tests/test_views.py

```diff
@@ -0,0 +1,1050 @@
+        request = rf.get('/')
+        response = views.nomination_about(request)
+        assert response.status_code == 200
+
+    def test_template_used(self, client):
+        response = client.get(reverse('nomination_about'))
+        assert response.templates[0].name == 'nomination/about.html'
+
+
+class TestNominationHelp():
+
+    def test_status_ok(self, rf):
+        request = rf.get('/')
+        response = views.nomination_help(request)
+        assert response.status_code == 200
+
+    def test_template_used(self, client):
+        response = client.get(reverse('nomination_help'))
+        assert response.templates[0].name == 'nomination/help.html'
+
+
+class TestUrlLookup():
+
+    def test_status_ok(self, rf):
+        project = factories.ProjectFactory()
+        url = factories.URLFactory(url_project=project, attribute='surt')
+        request = rf.post('/', {'search-url-value': url.entity, 'partial-search': ''})
+        response = views.url_lookup(request, project.project_slug)
+
+        assert response.status_code == 200
+
+    def test_redirects(self, rf):
+        project = factories.ProjectFactory()
+        request = rf.post('/', {'search-url-value': 'a_url'})
+        response = views.url_lookup(request, project.project_slug)
+
+        assert response.status_code == 302
+        assert response['Location'] == '/nomination/{0}/url/a_url/'.format(project.project_slug)
+
+    def test_raises_http404_if_not_post(self, rf):
+        project = factories.ProjectFactory()
+        request = rf.get('/')
+
+        with pytest.raises(Http404):
+            views.url_lookup(request, project.project_slug)
+
+    def test_raises_http404_with_no_search_url_value(self, rf):
+        project = factories.ProjectFactory()
+        factories.URLFactory(url_project=project, attribute='surt')
+        request = rf.post('/', {'partial-search': ''})
+
+        with pytest.raises(Http404):
+            views.url_lookup(request, project.project_slug)
+
+    def test_template_used(self, client):
+        project = factories.ProjectFactory()
+        url = factories.URLFactory(url_project=project, attribute='surt')
+        response = client.post(
+            reverse('url_lookup', args=[project.project_slug]),
+            {'search-url-value': url.entity, 'partial-search': ''}
+        )
+
+        assert response.templates[0].name == 'nomination/url_search_results.html'
+
+    def test_context(self, client):
+        project = factories.ProjectFactory()
+        url = factories.URLFactory(url_project=project, attribute='surt')
+        response = client.post(
+            reverse('url_lookup', args=[project.project_slug]),
+            {'search-url-value': url.entity, 'partial-search': ''}
+        )
+
+        assert response.context['project'] == project
+        assert len(response.context['url_list']) == 1
+        assert response.context['url_list'][0] == url
+
+
+class TestProjectUrls():
+
+    @pytest.mark.parametrize('request_type', [
```

Comment:
What is the reasoning behind parametrizing this? I am not confident this is really that useful.